### DISCUSSION TITLE
Google signin and signup problem

### DISCUSSION THREAD

**Discussion by rahid780** [TheAlphamerc/flutter_twitter_clone#297] [Q&A]
when i am continuing with google it giving me error and it create user but not store its information to firebase: 
![Untitled](https://github.com/TheAlphamerc/flutter_twitter_clone/assets/118201107/8c80ff37-b13a-410d-93ea-b71eedc65c3d)

**Comment by TheAlphamerc** [comment]
It may be because you are getting null check error.

**Reply by mohitdebian** [reply]
What's that

**Reply by TheAlphamerc** [reply]
It is stated in the screenshot.

**Reply by mohitdebian** [reply]
Ok, thanks
